cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
`view.backgroundColor = .white`
would look even more neat 😄

You can also use XCAssets as we might implement dark mode in the future.
I'll be changing it in a while, I'm implementing SwiftGen right now 😉